interface/patient_file/addr_label.php

```diff
@@ -48,37 +55,51 @@
 $today = oeFormatShortDate($date='today');
 $dob = oeFormatShortDate($patdata['DOB']);
 
-$pdf = new PDF_Label('5160'); // used this to get the basic info to the class
-$pdf = new eFPDF('P', 'mm',array(102,252)); // set the orentation, unit of measure and size of the page
-$pdf->AddPage();
-$pdf->SetFont('Arial','',50);
 
+//Format envelope settings.  Changes made by Daniel Pflieger
+//Pull the settings from globals.  If the Global settings have not been determined in Globals,
+//the default is to print on a 4 1/8 by 9 1/2 envelope.
+//todo: put the option in settings to set envelope size.
+
```

Comment:
Did you do away with just the labels? What if they are going on something besides an envelope? I know there was a change to the appointment report to allow labels.